base_geoengine/static/src/js/widgets/geoengine_widgets.js

```diff
@@ -56,7 +56,7 @@ var FieldGeoEngineEditMap = AbstractField.extend(geoengine_common.GeoengineMixin
         });
     },
 
-    create_layers: function(self, field_infos) {
+    create_layersAbstractFieldself, field_infos) {
```

Comment:
There's probably a missing ``: function(`` after ``create_layersAbstractField``

Reply:
This indeed seems to be a typo